libindy/src/services/pool/types.rs

```diff
@@ -465,6 +465,27 @@ impl PoolConfig {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct PoolOpenConfig {
+    #[serde(default)]
+    pub timeout: i64,
+    #[serde(default)]
+    pub extended_timeout: i64,
+}
+
+impl Default for PoolOpenConfig {
+    fn default() -> Self {
+        PoolOpenConfig {
+            timeout: super::networker::POOL_ACK_TIMEOUT,
+            extended_timeout: super::networker::POOL_REPLY_TIMEOUT,
+        }
+    }
+}
+
+impl JsonEncodable for PoolOpenConfig {}
```

Comment:
I suggest to avoid mixed state in master (partial removed `JsonEncodable`). If we decide to do something else with JsonEncodable against "just remove" it will be easy to replace all JsonEncodable at once.